cmd/vela-github-release/gh.go

```diff
@@ -17,7 +17,7 @@ import (
 const (
 	_gh       = "/bin/gh"
 	_ghTmp    = "/bin/download"
-	_download = "https://github.com/cli/cli/releases/download/v%s/gh_%s_%s_%s.tar.gz//gh_%s_%s_%s/bin"
+	_download = "https://github.com/cli/cli/releases/download/v%s/gh_%s_%s_%s.tar.gz/gh_%s_%s_%s/bin"
```

Comment:
Did we test this change to ensure it works?

I ask, because the `//` was intentional and at one point I know we needed it:

https://github.com/hashicorp/go-getter#subdirectories

Reply:
I'll confirm again. The test was failing previously on download (not due to this). But it seemed it work afterwards. It's possible I didn't test it well enough. Thanks.

Reply:
you were correct. i wasn't fully aware how go-getter worked. looks like `//` is a delimiter. it downloads from the url before it, automatically extracts and uses the info after `//` to copy what you want from the extracted contents. 👍🏼 

i reverted the change. thanks!